package.json

```diff
@@ -33,6 +34,10 @@
       {
         "command": "onevscode.configuration-settings",
         "title": "one configuration-settings"
+      },
+      {
+        "command": "onevscode.toggle-codelens",
+        "title": "toggle codelens"
```

Comment:
other commands start with `one`, to distinghish from other vscode-extensions commands, and should be filtered when user tyes `one`.
`toggle codelens` looks generic that doesn't reflect this is from `one vscode extension`
It would be better to use like `one toggle codelens or `one codelens toggle`

Reply:
How about category setting when setting command? Add One to the front as follows. 
This will be more effective in distinguishing it from other vcode-extensions commands.

`{
        "command": "onevscode.toggle-codelens",
        "title": "toggle codelens",
        "category": "One"
}`

![image](https://user-images.githubusercontent.com/44386047/135016163-33727fb7-2202-4edc-8212-8b9c8baa72d0.png)

If not, I will change it with `one toggle codelens`.

Reply:
Using category looks good :) Thanks sharing this!
I think using category seems a bit different from this PR's context so, could you please post another PR to change all existing after this PR lands ?

Reply:
We use `ONE` in all capitals :)

Reply:
> In the current PR, can I apply category and upload it?

As that would use different convention to existing ones, it's not recommended. So I prefer to be in another PR.

> Can I remove all of the 'one' in the title and move it to the category?

Yes, that is what I suggest, but may need to get reviews for others too.